docs/reference/release-notes/highlights-7.7.0.asciidoc

```diff
@@ -18,3 +18,110 @@ pivot and summarize your data and store it in a new index. See
 {ref}//transform-apis.html[{transform-cap} APIs].
 
 // end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+=== Finer memory control for bucket aggregations
+
+A new setting has been introduced - `search.check_buckets_step_size` - to
+better control how the coordinating node will allocate memory when aggregating
+buckets. The allocation of buckets will now be done in steps, each step
+allocating a number of buckets equal to this setting. A parent circuit breaker
+check is performed on allocation, to avoid an OOM failure.
+
+// end::notable-highlights[]
+
+// tag::notable-highlights[]
+[float]
+=== A new way of searching: asynchronously
+
+Searches can now be submitted to a new endpoint, `_async_search`, that works
+exactly the same as the `_search` endpoint, except that instead of blocking and
+returning the final response only once it's entirely available, it will pan the
+results as they become available, asynchronously.
```

Comment:
Technically, "endpoint" is a more accurate term here but we use "API" throughout our docs.
```suggestion
You can now submit {ref}/async-search-intro.html[long-running searches] using
the new {ref}/async-search.html[`_async_search` API]. The new API accepts the
same parameters and request body as the {ref}/search-search.html[Search API].
However, instead of blocking and returning the final response only when it's 
entirely finished, you can retrieve results from an async search as they become 
available.
```

Reply:
> Technically, "endpoint" is a more accurate term here but we use "API" throughout our docs.

I've reused the PR phrasing in this case. @jrodewig, if you think this would be strongly dissonant  to the rest of the docs, we can change it. OTOH, this being just the highlights, we could also leave it as is.